custom_components/tahoma/__init__.py

```diff
@@ -86,7 +87,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     username = entry.data.get(CONF_USERNAME)
     password = entry.data.get(CONF_PASSWORD)
 
-    session = aiohttp_client.async_create_clientsession(hass)
+    session = async_get_clientsession(hass)
```

Comment:
don't we need to create a session at some point before doing a get? 
if not, feel free to merge!

Reply:
Nope HA does it for us. That's why it's nice! I tested locally to be sure.